src/addons/editors/CheckboxEditor.js

```diff
@@ -15,17 +15,14 @@ var CheckBoxEditor = React.createClass({
   },
 
   render(): ?ReactElement{
-    return (<input className="react-grid-CheckBox" type="checkbox" checked={this.props.value} onChange={this.handleChange} />);
+    var checked = this.props.value != null ? this.props.value : false;
+    return (<input className="react-grid-CheckBox" type="checkbox" checked={checked} onClick={this.handleChange} />);
   },
 
   handleChange(e: Event){
+    e.stopPropagation();
```

Comment:
do we always want to stop propagation? wouldnt it be better to possibly pass the event through to onRowSelect?